change/@fluentui-react-tabs-7eea6ae2-201d-49d4-8749-15add0838235.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "fix(react-tabs): fix missing dependencies and lint warnings",
```

Comment:
```suggestion
  "comment": "fix missing dependencies and lint warnings",
```

Reply:
right. for more context: I always use conventional commits which also makes it easy to identify what commit belongs to what change-file. I know its redundant in change-log. would be great to have tooling for that in place.